lib/cmds/opts.ts

```diff
@@ -42,9 +42,11 @@ var opts: Options = {};
 opts[OUT_DIR] = new Option(OUT_DIR, 'Location to output/expect', 'string', Config.getSeleniumDir());
 opts[SELENIUM_PORT] =
     new Option(SELENIUM_PORT, 'Optional port for the selenium standalone server', 'string', '4444');
-opts[APPIUM_PORT] = new Option(APPIUM_PORT, 'Optional port for the appium server', 'string');
+opts[APPIUM_PORT] =
+    new Option(APPIUM_PORT, 'Optional port for the appium server', 'string', '4723');
```

Comment:
In general, ports should be strings.  But arithmetic is done on the android port number, and it has to be between 5554 and 5586 anyway, so I decided to make it a number in this case.